README.md

```diff
@@ -46,6 +46,13 @@ The default model is Flag Embedding, which is top of the [MTEB](https://huggingf
 
 - [**prithivida/Splade_PP_en_v1**](https://huggingface.co/prithivida/Splade_PP_en_v1) - Default
 
+### Image Embedding
```

Comment:
Not that `TokenizerFiles` is removed, this [README.md section](https://github.com/Anush008/fastembed-rs#:~:text=results)%3B-,Alternatively%2C%20raw,-.onnx%20files%20can) needs to be updated since it's much simpler now.